vue/components/ui/atoms/dropdown/dropdown.vue

```diff
@@ -143,11 +143,17 @@
     width: auto;
 }
 
-.dropdown-container .dropdown > .dropdown-item:hover,
-.dropdown-container .dropdown > .dropdown-item:active,
+.dropdown-container .dropdown > .dropdown-item:not(.disabled):hover,
+.dropdown-container .dropdown > .dropdown-item:not(.disabled):active,
 .dropdown-container .dropdown > .dropdown-item.selected,
-.dropdown-container .dropdown > .dropdown-item.highlighted {
+.dropdown-container .dropdown > .dropdown-item:not(.disabled).highlighted {
     background-color: $soft-blue;
+    font-weight: 700;
+}
+
+.dropdown-container .dropdown > .dropdown-item.disabled {
+    cursor: not-allowed;

```

Comment:
let's just use the regular cursor like this example
https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_option_disabled